dspace/config/spring/api/virtual-metadata.xml.openaire4

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why does this file end in an `.openaire4` extension?  Is it to disable this by default?  If so, I wonder if there's an easier way to handle disabling these beans by default.  I'm not sure of how to do this myself, but we could look into perhaps doing it via some form of configuration (as all `dspace.cfg` configs are available in Spring XML configs) if there's a way to do that in Spring.

Reply:
Although I'd consider it out of scope for this PR, one of the solutions I've considered in the past for creating separate virtual metadata configuration per model would be based on the concept of importing spring configuration
We could perhaps use an empty configuration by default, and support importing the openaire4, or alternative models

Reply:
Yes @tdonohue. I'm also not so fun of this solution. And if changing the default virtual metatada configuration file, I may be introducing load of unnecessary data. So the idea here is to have an alternative configuration file, that isn't processed when loading spring beans and can be used (replacing the existing file virtual-metadata.xml) to enhance and to return more openAIRE4 data (otherwise you will have the bare minimum).
As referred in this comment https://github.com/DSpace/DSpace/pull/2592#issuecomment-566099897 an improvement could be done to Virtual Metadata to address the need of having different configurations for each "media" to be used. It could be used a specific configuration in the angular context and a different one for OAI meaning according the the "presentation media" we could use different configurations.

Reply:
I agree with @benbosman. Changing the virtual metadata could be addressed in a different PR and I'm willing to help on that. And if this PR is merged with this file also change it to better suit the result of that change.